jekyll/_data/sidenav.yml

```diff
@@ -46,6 +46,14 @@ en:
 - name: Pipelines
   icon: icons/sidebar/pipelines.svg
   children:
+    - name: Overview
+      children:
+        - name: Pipelines overview
+          link: 2.0/pipelines/
+        - name: Pipeline processing
+          link: 2.0/build-processing
```

Comment:
Can we change the target of this link? Can we add a redirect from the old value? (Same for "skip build" below)

Reply:
Many of our URLs are incorrect/outdated, which is a separate issue we're thinking about. There's no easy fix right now due to SEO concerns I believe.